src/pick.js

```diff
@@ -9,7 +8,7 @@ var _indexOf = require('./internal/_indexOf');
  * @func
  * @memberOf R
  * @category Object
- * @sig [String] -> {String: *} -> {String: *}
+ * @sig [k] -> {k: v} -> {k: v}
```

Comment:
i don't have strong feelings on the subject. I will point out, however, that the whole point is that this implementation will support numeric keys. So `k` may be more accurate than `String`